src/Acuminator/Acuminator.Analyzers/StaticAnalysis/CallingBaseDataViewDelegate/CallingBaseDataViewDelegateFromOverrideDelegateAnalyzer.cs

```diff
@@ -0,0 +1,117 @@
+
+        public void Analyze(SymbolAnalysisContext context, PXContext pxContext, CodeAnalysisSettings settings, PXGraphSemanticModel pxGraph)
+        {
+            context.CancellationToken.ThrowIfCancellationRequested();
+
+            var ownDelegatesDictionary = pxGraph.ViewDelegates
+                                         .Where(d => pxGraph.Symbol.Equals(d.Symbol.ContainingType))
+                                         .ToDictionary(d => d.Symbol.Name, d => d, StringComparer.OrdinalIgnoreCase);
+            var ownRelatedViews = pxGraph.Views
+                                  .Where(v => ownDelegatesDictionary.ContainsKey(v.Symbol.Name) &&
+                                              pxGraph.Symbol.Equals(v.Symbol.ContainingType))
+                                  .ToArray();
+            var baseNonRedeclaredRelatedViews = new List<DataViewInfo>();
+
+            foreach (var view in pxGraph.Views)
+            {
+                for (var curView = view; curView != null; curView = curView.Base)
+                {
+                    if (ownDelegatesDictionary.ContainsKey(curView.Symbol.Name) &&
+                        !pxGraph.Symbol.Equals(curView.Symbol.ContainingType) &&
+                        ownRelatedViews.All(v => !v.Symbol.Name.Equals(curView.Symbol.Name, StringComparison.OrdinalIgnoreCase)))
+                    {
+                        baseNonRedeclaredRelatedViews.Add(curView);
+                    }
+                }
+            }
+
+            foreach (var viewDelegate in ownDelegatesDictionary.Values)
+            {
+                var walker = new Walker(context, pxContext, baseNonRedeclaredRelatedViews);
+
+                walker.Visit(viewDelegate.Node);
+            }
+        }
+
+        private class Walker : NestedInvocationWalker
+        {
+            private readonly SymbolAnalysisContext _context;
+            private readonly PXContext _pxContext;
+            private readonly IEnumerable<DataViewInfo> _nonRedeclaredBaseViews;
+
+            public Walker(SymbolAnalysisContext context, PXContext pxContext, IEnumerable<DataViewInfo> nonRedeclaredBaseViews)
+                : base(context.Compilation, context.CancellationToken)
+            {
+                pxContext.ThrowOnNull();
+                nonRedeclaredBaseViews.ThrowOnNull();
+
+                _context = context;
+                _pxContext = pxContext;
+                _nonRedeclaredBaseViews = nonRedeclaredBaseViews;
+            }
+
+            public override void VisitMemberAccessExpression(MemberAccessExpressionSyntax node)
+            {
+                ThrowIfCancellationRequested();
+                _context.CancellationToken.ThrowIfCancellationRequested();
+
+                var reported = false;
+                var symbol = GetSymbol<ISymbol>(node);
+                var expressionSymbol = GetSymbol<ISymbol>(node.Expression);
```

Comment:
I think that this check should be similar to other walkers - get `OverrideDefinition` from `OriginalDefinition` because you can skip some method symbols otherwise

Reply:
Ok